mage/algorithms/traditional-graph-analytics/max-flow.md

```diff
@@ -0,0 +1,43 @@
+---
+id: maximum-flow-algorithm
+title: Maximum Flow
+sidebar_label: Maximum Flow
+---
+
+## Description
+
+Maximum Flow problem in optimization theory regards finding the **maximum
+possible flow** going through a flow network from source to sink nodes. A **flow
+network**, or a transportation network, is a directed graph with edge weights
+representing flow capacity. The incoming flow must equal outgoing flow for every
+node, except the source and sink nodes. Such networks can be used to model
+computer networks, fluids in pipes, currents in an electrical circuit, road and
+railway networks.
+
+Having a variety of applications, max flow can be used for maximum matching,
+edge-disjoint paths and node-disjoint paths, to name a few. The max-flow min-cut
+theorem states that having found a maximum flow of a graph, we can also find its
+minimum cut.
+
+Multiple algorithms exist for solving the Maximum Flow problem, such as the
+Ford-Fulkerson method, Edmonds-Karp and push-relabel algorithm.
+
+<img src={require('../../data/algorithms/traditional-graph-analytics/maximum-flow-algorithm.png').default} alt="maximum-flow-algorithm" width="600"/>
+
+> Example of augmenting paths in max flow with edge property -> {flow /
+> capacity}, max flow is 19, the sum of inflows to sink (or outflows from
+> source)
+
+## Materials
+
+### Implementation
+
+[![Maximum
+Flow](https://img.shields.io/badge/Maximum_Flow-Implementation-FB6E00?style=for-the-badge&logo=github&logoColor=white)](https://github.com/memgraph/mage/blob/main/python/max_flow.py)
+
```

Comment:
If  you still have drawio file for the graph you've made, could you maybe adapt it to Memgraph colors?